src/OmniSharp.Host/project.json

```diff
@@ -8,36 +8,35 @@
     "OmniSharp.Stdio": "1.0.0",
     "OmniSharp.Plugins": "1.0.0",
     "OmniSharp.Roslyn": "1.0.0",
-    "Microsoft.AspNetCore.Diagnostics": "1.0.0",
-    "Microsoft.AspNetCore.Server.Kestrel": "1.0.0",
+    "Microsoft.AspNetCore.Diagnostics": "1.1.0",
```

Comment:
I *really* want to invert OmniSharp at some point and build the HTTP endpoints on top of the stdio. Make Http a plugin so that core OmniSharp is lean and mean.